.github/workflows/adhoc-prerelease.yml

```
@@ -1,13 +1,16 @@
-name: Pages
+name: Adhoc Prerelease
 
 on:
-  push:
-    branches:
-      - pages
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: "Tag"
+        required: false
```

maybe this should mandatory? We'll never want to do a production release with this, yes?
```suggestion
        required: true
```

---

Good point!

---

I did think about some of this, but anyone in the npm org could just publish locally if they wanted. So I figure we trust everyone who is a maintainer of this repo?

Yeah, the primary reason this arose was that I wanted to test more than one thing, so kind of out-grew the Pages workflow. Could definitely keep it if you're happy to start amassing several branches/tags in that though.

---

Can this workflow only be run by maintainers? (I actually dunno). That could work. But we should still fail the flow if the input is `latest`, `beta` or `alpha` imo. 

Let's keep the Pages workflow for now and delete once we don't use it for a while, fair? Which is to say, once folks stop making commits there (or is that just you rn?)